util/crc32c.cc

```diff
@@ -9,12 +9,12 @@
 //
 // A portable implementation of crc32c, optimized to handle
 // four bytes at a time.
-
+#include <iostream>
 #include "util/crc32c.h"
-
 #include <stdint.h>
 #ifdef HAVE_SSE42
 #include <nmmintrin.h>
+#include <wmmintrin.h>
```

Comment:
I think we need -maes now, not just -msse4.2. We can put a check in `build_tools/build_detect_platform` and set a new flag similarly to how we set `-DHAVE_SSE42`. And then use 3-way crc if that new flag is set and `NO_THREEWAY_CRC32C` is not set.

Reply:
Where are we using AES? I don't see any AES intrinsics in RocksDB.

Reply:
just the carryless multiplication (_mm_clmulepi64_si128) is grouped together with `-maes`, although I now realize it can be enabled more specifically with `-mpclmul`. You can repro the issue like this:

`make clean && PORTABLE=1 USE_SSE=1 make -j64 util/crc32c.o`